functions/hw_7_func.py

```diff
@@ -0,0 +1,41 @@
+import sys
+
+def sequential_map (*args):
+    *funcs , numbers = args
+    for func in funcs:
+        cont = list(map(func, numbers))
+    return cont

```

Comment:
1. Превратить элемент в список можно только в самом конце, результат map можно передавать в другие map'ы
2. Твоя функция возвращает только результат применения последней функции из переданных
```suggestion
def sequential_map (*args):
    *funcs , numbers = args
    for func in funcs:
        numbers = map(func, numbers)
    return list(numbers)

```